src/commands/preload/index.ts

```diff
@@ -295,7 +295,7 @@ Can be repeated to add multiple certificates.\
 		owns__release: {
 			$select: ['id', 'commit', 'end_timestamp', 'composition'],
 			$expand: {
-				contains__image: {
+				release_image: {
```

Comment:
Please double check that this works, since I don't see any related code that consumes this result being updated.

Reply:
Seems to me this is used in `balena-preload`. Opened a PR updating the SDK and addressing breaking changes there as well: https://github.com/balena-io-modules/balena-preload/pull/299